lib/streamlit/DeltaGenerator.py

```diff
@@ -619,6 +628,10 @@ def dataframe(self, _, data=None):
             values and colors. (It does not support some of the more exotic
             pandas styling features, like bar charts, hovering, and captions.)
             Styler support is experimental!
+        width : int
+            Desired width of the UI element expressed in CSS points
```

Comment:
Replace: points → pixels

Also mention that when set to `None` we will calculate an appropriate width for you.

Same below